project/CONTRIBUTING.md

```diff
@@ -130,6 +130,32 @@ Notes:
     * If your BC PR is accepted, you can do a new one on the `{{ unstable_branch }}` branch which remove the deprecated code.
     * SYMFONY DOC REF (same logic)?
 
+If you have a non-BC PR to propose, please first try to create a related BC PR first.
+This BC PR should mark every piece of code that needs to be removed / uncommented / reworked
+in the corresponding non-BC PR with the following marker comment : `TODO_NEXT_MAJOR`.
+When the BC PR is merged in the stable branch, wait for the stable branch to be
+merged in the unstable branch, and then work on your non-BC PR.
+
+For instance, assuming you want to introduce a new method to an existing interface, you should do something like this:
+
+```php
+<?php
+
+namespace Foo;
+
+interface BarInterface
+{
+
+  // TODO_NEXT_MAJOR : uncomment this method
```

Comment:
* `TODO_NEXT_MAJOR :` -> `TODO_NEXT_MAJOR:`
* I thinkg `NEXT_MAJOR:` is enough.